formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -200,10 +200,9 @@ type variantType =
 
 [@@onVariantType]
 type gadtType('x) =
-  | Foo(int) : [@onFirstRow] gadtType(int)
-  | Bar
-      ([@onInt] int)
-      : [@onSecondRow] gadtType(unit)
+  | Foo(int): [@onFirstRow] gadtType(int)
```

Comment:
I actually could have gone either way on this one. It did sometimes bother me how before the GADT colon was stuck to the constructor.